contracts/bid/Cargo.toml

```diff
@@ -8,26 +8,24 @@ edition = "2018"
 crate-type = ["cdylib", "rlib"]
 
 [dependencies]
-dusk-abi = "0.5"
-canonical = {version="0.5", default-features=false}
-canonical_derive = "0.5"
-rusk-abi = {path = "../../rusk-abi"}
+dusk-abi = "0.9.0-rc"
+canonical = { version="0.6", default-features=false }
+canonical_derive = "0.6"
+rusk-abi = { path = "../../rusk-abi" }
 dusk-bytes = "0.1"
-dusk-bls12_381 = {version="0.6", features=["canon"], default-features=false}
-dusk-jubjub = {version="0.8", features=["canon"], default-features=false}
-dusk-poseidon = {version="0.18", features=["canon"], default-features=false}
-dusk-pki = {version = "0.6", features = ["canon"]}
-phoenix-core = {git="https://github.com/dusk-network/phoenix-core", tag="v0.9.1", features = ["canon"]}
-dusk-kelvin-map = "0.3"
-nstack = "0.7"
-microkelvin = "0.6"
-schnorr = {git="https://github.com/dusk-network/schnorr", tag="v0.5.1", default-features = false, features = ["canon"]}
-dusk-blindbid = {version = "0.7", default-features = false, features = ["canon"]}
+dusk-bls12_381 = { version="0.8", features=["canon"], default-features=false }
+dusk-jubjub = { version="0.10", features=["canon"], default-features=false }
+dusk-poseidon = { version="0.21.0-rc", features=["canon"], default-features=false }
+dusk-pki = { version = "0.7.0-rc", default-features = true,  features = ["canon"] }
+phoenix-core = { version = "0.11.0-rc", default-features = true, features = ["canon"] }
+dusk-hamt = "0.3"
+nstack = "0.8"
+microkelvin = "0.7"
+dusk-schnorr = { version = "0.7.0-rc", default-features = false, features = ["canon"] }
+dusk-blindbid = { version = "0.8.0-rc", default-features = false, features = ["canon"] }
 
 # Test dependencies
-rand = {version="0.7", optional=true}
-rusk-vm = {git="https://github.com/dusk-network/rusk-vm", tag="v0.4.0", optional=true}
-canonical_host = {version="0.5", optional=true}
+rusk-vm = { version = "0.6.0-rc", optional = true }
 
 [features]
-host=["rand", "rusk-vm", "canonical_host"]
+host=["rusk-vm"]
```

Comment:
I believe the host/hosted features concept is outdated to latest canonical structure

Reply:
Addressed